pkg/log/log.go

```diff
@@ -3,8 +3,8 @@ package log
 import (
 	"strings"
 
-	"github.com/Sirupsen/logrus"
 	"github.com/fatih/color"
+	"github.com/sirupsen/logrus"
```

Comment:
does it work with case change from upper case S to lower?

Reply:
Yes, it needs to be lower case now.

Reference: https://github.com/sirupsen/logrus/issues/570#issuecomment-313933276